scripts/lib/install

```diff
@@ -381,6 +381,7 @@ if [ "$VIRTUALENV_NEEDED" = "yes" ]; then
 fi
 
 "$ZULIP_PATH"/scripts/lib/install-node
+"$ZULIP_PATH"/scripts/lib/install-yarn
```

Comment:
We need this in `scripts/lib/upgrade-zulip-stage-2` too, right?

Reply:
Oops! Fixed.